doc/design.rst

```diff
@@ -193,6 +193,33 @@ _________________
       - Takes a JSON of known vulnerabilities affecting a repo & creates GitHub Issues
         in said repo for any vulnerabilities, which do not yet have a GitHub Issue
 
+security-issues
+^^^^^^^^^^^^^^^
+The `security-issues/action.yml` creates GitHub Issues for known vulnerabilities
+for `maven` and `pip-audit`.
+The following steps are taken:
+
+1. Convert a JSON of known vulnerabilities into a common format (`class Issue`)
+2. Filter out vulnerabilities which already have an existing GitHub Issue via CVE
+3. Create new GitHub Issues
+4. Return a JSON of the newly created GitHub Issues
```

Comment:
Very good, thank you so much!

Reply:
Could please also add a note on the different JSON dialects for maven vs. `pip-audit`, here? 
Please also add a reference to the definition (at least for pip-audit): 
> https://pypi.org/project/pip-audit/, search for "Audit dependencies in JSON format".